ray-on-gke/README.md

```diff
@@ -1,130 +1,4 @@
-# Ray on GKE
+# Running Flyte on GKE
```

Comment:
Why did this change to "Running Flyte on GKE"?

Reply:
Typo - Just fixed it